pom.xml

```diff
@@ -65,6 +65,33 @@
       <version>2.9.0</version>
     </dependency>
 
+    <!--<dependency>-->
+      <!--<groupId>io.opentracing</groupId>-->
+      <!--<artifactId>opentracing-api</artifactId>-->
+      <!--<version>0.33.0</version>-->
+    <!--</dependency>-->
+
+    <!-- OpenTracing API -->
+    <dependency>
+      <groupId>io.opentracing</groupId>
+      <artifactId>opentracing-api</artifactId>
+      <version>0.31.0</version>
+    </dependency>
+
+    <!-- OpenTracing Util -->
+    <dependency>
+      <groupId>io.opentracing</groupId>
+      <artifactId>opentracing-util</artifactId>
+      <version>0.31.0</version>
```

Comment:
Our docs are out of date... please use `0.33.0` instead.

These are also explicit dependencies from `dd-trace-ot`, so they could be omitted if you like.